js/main.js

```diff
@@ -21,6 +28,80 @@ $form.addEventListener('submit', event => {
   };
   data.nextEntryId++;
   data.entries.unshift(obj);
+  $unorderedList.prepend(renderEntry(obj));
+  viewSwap('entries');
   $image.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form.reset();
 });
+
+function renderEntry(entry) {
+  const title = entry.title;
+  const notes = entry.notes;
+  const url = entry.url;
+  const $li = document.createElement('li');
+  const $rowDiv = document.createElement('div');
+  $rowDiv.setAttribute('class', 'row');
+  $li.appendChild($rowDiv);
+
+  // first column
+  const $imgCol = document.createElement('div');
+  $imgCol.setAttribute('class', 'column-half');
+  $rowDiv.appendChild($imgCol);
+
+  const $imgWrap = document.createElement('div');
+  $imgWrap.setAttribute('class', 'img-wrapper');
+  $imgCol.appendChild($imgWrap);
+
+  const $img = document.createElement('img');
+  $img.setAttribute('src', url);
+  $imgWrap.appendChild($img);
+
+  // second column
+  const $infoCol = document.createElement('div');
+  $infoCol.setAttribute('class', 'column-half');
+  $rowDiv.appendChild($infoCol);
+
+  const $title = document.createElement('h2');
+  $title.setAttribute('class', 'entry-title');
+  $title.textContent = title;
+  $infoCol.appendChild($title);
+
+  const $notes = document.createElement('p');
+  $notes.textContent = notes;
+  $infoCol.appendChild($notes);
+
+  return $li;
+}
+
+document.addEventListener('DOMContentLoaded', event => {
+  for (let i = 0; i < data.entries.length; i++) {
+    const currEntry = data.entries[i];
+    $unorderedList.appendChild(renderEntry(currEntry));
+  }
+  viewSwap(data.view);
+});
+
+function HideNoEntries() {
```

Comment:
Function names should be lowercase. You'll pretty much only use uppercase once we get to React, or if you are creating a `Class` in JS